report_form_saver.py

```diff
@@ -193,33 +194,60 @@ def validate_email(self, email):
             flash(f"Email ({email}) is not valid. Please enter a valid email address.")
             return False
 
+    @staticmethod
+    def make_tz_aware(time):
+        default_tz = pendulum.timezone(conf.get("core", "default_timezone"))
+        time_of_day_to_local = pendulum.datetime(
```

Comment:
How does this work when the timezone change would switch the date?  For example, 11PM pacific time is 2AM eastern on the next day.

Reply:
daily schedule it goes from 11pm to 2am daily. 

weekly schedule it will go from sunday 11pm to monday 2pm.

Reply:
So the weekly schedule doesn't use `make_tz_aware`?  Where is the weekly schedule timezone function?

Reply:
oh it's in the javascript. this is just the backend function that's passing the utc time to front end

Reply:
the time that we're storing in schedule_time isn't timezone aware so i'm making it timezone aware

Reply:
report_saver -> makes timezone aware and sets it as a value in the form
report.js -> takes the utc time from the elem value and converts to the local time

Reply:
OK, can you document that in the code?  It's not obvious if you don't know that you have to look in the Javascript for the other piece of the puzzle.